deploy/lib/Help.rb

```diff
@@ -266,6 +268,32 @@ def self.deploy
     DOC
   end
 
+  def self.merge
+    <<-DOC.strip_heredoc
+      Usage: ml {env} merge WHAT [options]
+
+      General options:
+        -v, [--verbose]  # Verbose output
+
+      Please choose a WHAT below.
+
+        content     # Merges your content db in the given environment
```

Comment:
Yes, likely. I could add modules, schemas, triggers like for deploy, but I didn't see much need for that. I can add anyhow..